src/Nancy.Siren.Demo/Infrastructure/OrderItemViewModelWriter.cs

```diff
@@ -33,7 +33,7 @@ public Siren Write(IEnumerable<OrderItemViewModel> data, Uri uri)
 
            
 
-            sirenDoc.links = new List<Link> { new Link { href = uri + "/", rel = new[] { "self" } }, new Link{href = uri.ToString().Replace("items",""), rel = new []{"order"}} };
+            sirenDoc.links = new List<Link> { new Link { href = uri.ToString() , rel = new[] { "self" } }, new Link{href = uri.ToString().Replace("/items",""), rel = new []{"order"}} };  

```

Comment:
What's the reason for making this ToString() and removing ToString() in `OrderWriter` ? 😄